index.html

```diff
@@ -246,6 +246,66 @@ <h3>
           </ul>
         </div>
       </section>
+
+      <section class="contact-container">
+        <div class="contact-text">
+          <h3>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h3>
+        </div>
+
+        <form
+          class="mobile-screen-form"
+          action="https://formspree.io/f/mzbovngz"
+          method="POST"
+        >
+          <ul>
+            <li>
+              <input
+                type="text"
+                name="username"
+                placeholder="Full name"
+                required
+                maxlength="30"
+              />
+            </li>
+            <li>
+              <input
+                type="email"
+                name="email"
+                placeholder="Email address"
+                required
+              />
+            </li>
+
+            <li>
+              <textarea
+                rows="10"
+                cols="30"
+                id="message-mobile"
+                name="usermessage"
+                maxlength="500"
+                required
+              >
+Enter Text here
+              </textarea>
+            </li>
+          </ul>
+          <button class="contact-button" type="submit">Get in touch</button>
+        </form>
+      </section>
+
+      <footer>
+        <ul class="footer-item">
+          <li><img src="Icons/social2/github.png" alt="Github icon" /></li>
+          <li><img src="Icons/social2/linkedin.png" alt="linkedin" /></li>
+          <li><img src="Icons/social2/angellist.png" alt="Angellist" /></li>
+          <li><img src="Icons/social2/twitter.png" alt="twitter" /></li>
+          <li><img src="Icons/social2/medium.png" alt="Meduim" /></li>
+        </ul>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Kindly use `a` tag inside the `li` tag.

Reply:
Good observation,
Noted!